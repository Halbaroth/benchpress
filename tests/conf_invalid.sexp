
; read smtlib status
(prover
  (name smtlib-read-status)
  (cmd "grep :status $file")
  (unknown ":status unknown")
  (sat ":status sat")
  (unsat ":status unsat"))

(prover
  (name minisat)
  (unsat "UNSATISFIABLE")
  (sat "))) ^SATISFIABLE")
  (cmd "minisat -cpu-lim=$timeout $file"))

(prover
  (name z3)
  (cmd "z3 $file")
  (version "cmd:z3 --version")
  (unsat "unsat|(s UNSATISFIABLE)")
  ( a b cc)
  (sat "(^sat)|(s SATISFIABLE)"))

(prover
  (name cvc4)
  (cmd "cvc4 $file ")
  (version "cmd:cvc4 --version | head -n 3")
  (yolo swag)
  (unsat "unsat")
  (sat "^sat"))

(task
  (name foo)
  (action
    (run_provers
      (dirs .)
      (provers cvc4))))
