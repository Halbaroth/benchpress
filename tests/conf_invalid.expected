
[31;1mError[0m:
  expected a valid Perl regex

  12 | 
  13 | ...... "))) ^SATISFIABLE".
  ------------------------------------------------------------
[34;1mContext[0m:
  expected a stanza

   9 | 
  10 | (prover
  ...
  13 |   (sat "))) ^SATISFIABLE")
  14 |   (cmd "minisat -cpu-lim=$timeout $file"))
  
